src/main/java/jenkins/plugins/publish_over_ssh/BapSshClient.java

```diff
@@ -39,9 +39,8 @@
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 
-import java.io.InputStream;
-import java.util.Stack;
-import java.util.Vector;
+import java.io.*;
+import java.util.*;
```

Comment:
Sorry this took so long to get back to. Please rework this so that * imports are not used.